modules/auxiliary/gather/zoomeye_search.rb

```diff
@@ -41,7 +44,7 @@ def initialize(info = {})
     register_options(
       [
         OptString.new('ZOOMEYE_APIKEY', [true, 'The ZoomEye api key']),
-        OptString.new('ZOOMEYE_DORK', [true, 'The ZoomEye dork']),
+        OptString.new('QUERY', [true, 'The ZoomEye dork']),
```

Comment:
Let's revert this change and put the datastore back to `ZOOMEYE_DORK`. There's two reasons for this:
1. The datastore option is more explicit as to what it is while `QUERY` is kind of generic
2. It was originally `ZOOMEYE_DORK` so changing the name of the option would alter automation that's been built around this without adding additional options to the registration